playground/hmr-ssr/__tests__/hmr-ssr.spec.ts

```diff
@@ -851,13 +851,16 @@ if (!isBuild) {
     editFile('self-accept-within-circular/c.js', (code) =>
       code.replace(`export const c = 'c'`, `export const c = 'cc'`),
     )
+    // it throws a same error as browser case,
+    // but it doesn't auto reload and it calls `hot.accept` called with `undefined`
+    await untilUpdated(() => el(), '')
```

Comment:
It turns out the error now became same as `playground/hmr` but module runner doesn't auto-reload, so the existing test failed.

```
 ReferenceError: Cannot access 'c' before initialization
```